src/qt/votingdialog.cpp

```diff
@@ -12,7 +12,11 @@
 	#include <QtCharts/QPieSeries>
 #endif
 
-#include <QtConcurrentRun>
+#if QT_VERSION < 0x050000
+    #include <QtConcurrentRun>
+#else
+    #include <QtConcurrent/QtConcurrentRun>
```

Comment:
Is this really the way to do it? When using qmake it automatically appends the correct include path (although with `-isystem` for some reason) so you don't have to specify the module prefix:

```
-isystem /usr/include/x86_64-linux-gnu/qt5/QtConcurrent
```

This should be the same for both Qt4 and 5 IIRC.